run/rsync_archive/copy-music.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -eu
+
+SRC="${RSYNC_USER}@${RSYNC_SERVER}:${MUSIC_SHARE_NAME}"
+DST="${MUSIC_MOUNT}"
+LOG="/tmp/rsyncmusiclog.txt"
+
+# check that DST is the mounted disk image, not the mountpoint directory
+if ! findmnt --mountpoint "$DST" > /dev/null
+then
+  log "$DST not mounted, skipping music sync"
+  exit
+fi
+
+function connectionmonitor {
+  while true
+  do
+    for _ in {1..10}
+    do
+      if timeout 3 /root/bin/archive-is-reachable.sh "$ARCHIVE_SERVER"
```

Comment:
This should either use `RSYNC_SERVER`, or this entire function should be removed and a timeout added to the rsync call, similar to what archive-clips.sh does.

Reply:
But `ARCHIVE_SERVER` is set to `RSYNC_SERVER` in script `archiveloop`, if in config `ARCHIVE_SYSTEM` is set to `rsync`. So this is "generic" and works as expected.